app/extensions/brave/ethwallet-popup.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const ipc = window.chrome.ipcRenderer
+let batAddress = null
+const indexUrl = `${window.location.origin}/index.html`
+
+ipc.send('get-popup-bat-balance')
```

Comment:
we don't have a good way to do state updates in extensions with redux so an ipc message is ok, but for sending messages to the browser process we should be using actions which do work in extensions

Reply:
the problem is sending the message back to the popup window, which is a browserAction popup and not in the redux appState. is there a way to get the correct webcontents in reducers using `event.sender.getId()` if the sender isn't a redux tab?